### DISCUSSION TITLE
problem with new SPOTREP 00102 update on nitrado server

### DISCUSSION THREAD

**Discussion by ocibisi** [Vdauphin/HeartsAndMinds#1327] [Question and answer]
Hello and congratulations on your creation. Unfortunately after the spotrep 00102 update starting the mission in nitrado's personal server does not work. Indeed it honestly does restart the server. I have verified that it is not a server problem but it concerns something else and they indicate the mission to me. I am using the latest version by chance this problem is encountered by you too?

**Comment by Vdauphin** [comment]
Hello,
Thank you
No bug here and same for those comunities https://www.battlemetrics.com/servers/arma3?q=hearts%2Band%2Bminds%20||%20%22H%26M%22&status=online&sort=-players
it is allways the same thing, as it is said in the arma news verify your game data and remove garbage mods
Cheers

**Reply by ocibisi** [reply]
Thanks a lot for the answer. I'm trying. Your mission is for me one of the most beautiful and I would be very sorry not to be able to play it anymore. YES it is true we play with many mods ... but until the update everything worked. Also I tried to start another Liberation mission ... and there the server works. I don't know where to look anymore. Can I ask you if by any chance the saves of the Database Saved mission are saved in a specific area of the server? I don't want it to be a conflict. But I honestly don't know where to look. I know that it is not easy to advise me but if you have a way to give me some info, thank you. Congratulations again for creating the most beautiful and expansive weapon 3 mission for me.

**Reply by Vdauphin** [reply]
> but until the update everything worked

Just ask to the mod author to fix his mod 

> Can I ask you if by any chance the saves of the Database Saved mission are saved in a specific area of the server? 

*.vars.Arma3Profile see https://community.bistudio.com/wiki/profileNamespace and https://community.bistudio.com/wiki/Profile

> But I honestly don't know where to look. I know that it is not easy to advise me but if you have a way to give me some info, thank you.

Join the Arma 3 discorde,  there are plenty of people open to help you with server bug
But the first thing is to do what I told you, remove one by one each mod until it works and check your rpt 

Cheers

**Reply by ocibisi** [reply]
thank you so much for your patience.
I am taking action as per your advice. I'm reloading all mods in the server and trying to find the mission database saves in the server ... but unfortunately I don't understand where they can be placed. I will try !.
As mentioned above, I thank you and renew the beauty of the mission. I also allow myself to ask you if I have any ideas, proposals to forward ... can I do it in this discussion or is there a dedicated area? I'm sorry but I'm new and don't know much GITHUB. I write with a translator and I really hope it will be understandable.
Have a nice day thanks ;-)

**Reply by Vdauphin** [reply]
> can I do it in this discussion or is there a dedicated area? 

Sure, https://github.com/Vdauphin/HeartsAndMinds/issues/738
